docs/DeveloperGuide.md

```diff
@@ -432,7 +432,7 @@ persons with the specified module field, it will return an error to the user.
 
 </div>
 
-The following sequence diagram shows how the `removeall` command is executed:
+The following sequence diagram shows how the `clean` command is executed:
 <p align="center">
   <img src="images/GiveSequenceDiagram.png">
```

Comment:
but actually, the below is give command? need to replace the diagram

Reply:
done